openid_connect.server_conf

```diff
@@ -66,6 +69,70 @@
         error_page 500 502 504 @oidc_error;
     }
 
+    #
+    # User information endpoint for the following purposes:
+    # - Browser to periodically check if your are signed-in based on status code.
```

Comment:
How can the browser be forced to do this? Using custom JS code? What happens if we get a 401 Unauthorized, what does that mean? Access Token not valid? Whether it can be updated using a Refresh Token? I'm not sure that such tasks should be solved through the User Agent.

Reply:
- When we get `401` from `/userinfo` or customer's endpoints because access token isn't valid, User Agent can call `/login` to relogin or refresh token based on the status of IdP's token expiration.
- We need to separately discuss this item which I wanted to talk to you. I will show you the use case later.
- Hence, I will remove this comment to avoid confusion as for now.